File: Sources/gestion-culturelle-front/project/src/app/model/invitationDto.ts
import { ManifestationDto } from './manifestationDto';
import { User } from './user';


export class InvitationDto {
    id: number;
    manifestation: ManifestationDto;
    vip:User;
    reponse:String;
    dateInvitation:Date;


    constructor(id?: number,
        manifestation?: ManifestationDto,
        vip?:User,
        reponse?:String,
        dateInvitation?:Date) {
        this.id = id;
        this.manifestation = manifestation;
        this.vip=vip;
        this.reponse=reponse;
        this.dateInvitation=dateInvitation

    }
}

File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/InvitationDto.java
package com.afpa.cda.dto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class InvitationDto {

	private int id;
	
	private ManifestationDto manifestation;
	private UserDto vip;
	
	String reponse;
	Date dateInvitation;
	
}
